src/ardupilot_vehicle.cpp

```diff
@@ -1371,7 +1373,20 @@ Ardupilot_vehicle::Vehicle_command_act::Enable()
         }
     }
     command_count = cmd_messages.size();
-    Try();
+    if (!lazy_execution) {
+        Try();
+    } else {
+        if (lazy_timer) {
+            lazy_timer->Cancel();
+        }
+
+        int delay = 10;
+        dynamic_cast<Ardupilot_vehicle&>(vehicle).updateConfig("vehicle.ec101.auto.wait", delay);
```

Comment:
(nits) When considering versatility, it is better to set the timer value for each command.
or If only auto command, names like flag variable `is_auto_command` are better than `lazy_execution`.